src/pulumi.ts

```diff
@@ -0,0 +1,1604 @@
+              args: {
+                name: "stringArray",
+              },
+            },
+            {
+              name: "--secret",
+              description: "Marks a value as secret to be encrypted",
+              args: {
+                name: "stringArray",
+              },
+            },
+          ],
+        },
+      ],
+    },
+    {
+      name: "console",
+      description: "Opens the current stack in the Pulumi Console",
+      subcommands: [
+        ...inheritedOptions,
+        stackOption,
+        {
+          name: ["-h", "--help"],
+          description: "Help for console",
+        },
+      ],
+    },
+    {
+      name: "destroy",
+      description: "Destroy an existing stack and its resources",
+      subcommands: [
+        ...inheritedOptions,
+        ...upDestroyOptions,
+        yesOption,
+        skipPreview,
+        {
+          name: "--exclude-protected",
+          description:
+            "Do not destroy protected resources. Destroy all other resources",
+        },
+        {
+          name: ["-h", "--help"],
+          description: "Help for destroy",
+        },
+      ],
+    },
+    {
+      name: "import",
+      description: "Import resources into an existing stack",
+      subcommands: [
+        ...inheritedOptions,
+        configFileOption,
+        debugOption,
+        diffOption,
+        messageOption,
+        parallelOption,
+        stackOption,
+        yesOption,
+        skipPreview,
+        suppressPermalinkOption,
+        {
+          name: ["-f", "--file"],
+          description:
+            "The path to a JSON-encoded file containing a list of resources to import",
+          args: { name: "file" },
+        },
+        {
+          name: ["-o", "--out"],
+          description:
+            "The path to the file that will contain the generated resource declarations",
+          args: { name: "file" },
+        },
+        {
+          name: "--parent",
+          description:
+            "The name and URN of the parent resource in the format name=urn, where name is the variable name of the parent resource",
+          args: { name: "[name=urn]" },
+        },
+        {
+          name: "--protect",
+          description:
+            "Allow resources to be imported with protection from deletion enabled (default true)",
+        },
+        {
+          name: "--provider",
+          description:
+            "The name and URN of the provider to use for the import in the format name=urn, where name is the variable name for the provider resourceƒ",
+          args: { name: "[name=urn]" },
+        },
+        {
+          name: "--suppress-outputs",
+          description:
+            "Suppress display of stack outputs (in case they contain sensitive values)",
+        },
+        {
+          name: ["-h", "--help"],
+          description: "Help for import",
+        },
+      ],
+      args: { name: "[type] [name] [id]" },
```

Comment:
This should be split in multiple arguments:
```suggestion
      args: [
        {
          name: "type",
        },
        {
          name: "name",
        },
        ...
      ],
```

Reply:
done